internal/supervisor/installation.go

```diff
@@ -721,3 +763,41 @@ func (s *InstallationSupervisor) finalCreationTasks(installation *model.Installa
 	logger.Info("Finished final creation tasks")
 	return model.InstallationStateStable
 }
+
+// Helper funcs
+
+func (s *InstallationSupervisor) checkIfClusterInstallationsAreStable(installation *model.Installation, logger log.FieldLogger) (bool, error) {
+	clusterInstallations, err := s.store.GetClusterInstallations(&model.ClusterInstallationFilter{
+		InstallationID: installation.ID,
+		PerPage:        model.AllPerPage,
+	})
+	if err != nil {
+		logger.WithError(err).Warn("Failed to find cluster installations")
+		return false, nil
```

Comment:
Just curious why this is `false, nil` instead of `false, err`? Another way I could phrase the question would be: why is this error merely a warning? The reason is not immediately obvious to me.

Reply:
Ah, I meant to put a comment on that helper func to clear up that very question. Let me do that.